src/components/Navbar.jsx

```diff
@@ -0,0 +1,32 @@
+import { NavLink } from 'react-router-dom';
+
+const Navbar = () => {
+  const links = [
+    { path: '/', text: 'Home' },
+    { path: 'calculator', text: 'Calculator' },
+    { path: 'quote', text: 'Quote' },
+  ];
+
+  return (
+    <nav className="navbar">
+      <h1>Math Magician</h1>
+      <ul>
+        {links.map((link) => (
+          <li key={link.text}>
+            <NavLink
+              to={link.path}
+              style={({ isActive }) => ({
+                color: isActive ? 'yellowgreen' : undefined,
+              })}
+            >
+              {link.text}
+            </NavLink>
+
+          </li>
+        ))}
+      </ul>
+    </nav>
+  );
+};
+
```

Comment:
- [ ] Please make sure to add a border between the navbar menu options to follow the wireframe design.

### Your design

![image](https://github.com/BiancaMe/math-magicians/assets/83803414/7d59029a-7276-4d5d-b2d4-0cb469bdf217)

### Wireframe design

![image](https://github.com/BiancaMe/math-magicians/assets/83803414/c1961243-2ebf-4fbe-906d-ef1c48864302)